distribution/docs/src/main/resources/content/_managing/attributes-added-on-ingest.adoc

```diff
@@ -8,8 +8,8 @@
 == {title}
 
 A metacard is first created and populated by parsing the ingested resource with an <<_attributes_added_by_input_transformers,Input Transformer>>. +
-Then <<_attributes_added_by_attribute_injection,Attributes Are Injected>>, <<_developing_default_attribute_types,Default Attribute Types>> are applied, and <<_attributes_added_by_attribute_overrides_ingest,Attribute are Overridden>>. +
-Finally the metacard is passed through a series of <<_attributes_added_by_pre_authorization_plugins,Pre-Authorization Plugins>> and <<_attributes_added_by_pre_ingest_plugins,Pre-Ingest Plugins>>.
+Then <<{managing-prefix}attributes_added_by_attribute_injection,Attributes Are Injected>>, <<{developing-prefix}metacard_groomer,Default Attribute Types>> are applied, and <<_attributes_added_by_attribute_overrides_ingest,Attribute are Overridden>>. +
+Finally the metacard is passed through a series of <<_attributes_added_by{architecture-prefix}pre_authorization_plugins,Pre-Authorization Plugins>> and <<_attributes_added_by{architecture-prefix}pre_ingest_plugins,Pre-Ingest Plugins>>.
```

Comment:
❓ Should these have prefixes at the start of their links? Also same question about having prefixes in the middle of the text.

Reply:
yes. fixed.